<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CacheTab</class>
 <widget class="QWidget" name="CacheTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="vboxMain">
   <item>
    <widget class="QLabel" name="lblSysCache">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>If any image type settings were changed, you will need to
clear the system thumbnail cache.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnSysCache">
     <property name="text">
      <string>Clear the System Thumbnail Cache</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblRpCache">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>ROM Properties Page maintains its own download cache for
external images.
Clearing this cache will force external images to be
redownloaded.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnRpCache">
     <property name="text">
      <string>Clear the ROM Properties Page Download Cache</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="vspcCache">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="lblStatus">
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="RpProgressBar" name="pbStatus"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RpProgressBar</class>
   <extends>QProgressBar</extends>
   <header>config/RpProgressBar.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
